tests/floating_point_math/floating_point_math.fz

```diff
@@ -41,6 +41,14 @@ floating_point_math is
     zero := a.zero
     one := a.one
     two := one + one
+    three := two + one
+    four := two + two
```

Comment:
What are these for? 
For integers, I define two and ten since they are needed for conversion to / from string. For floats, this should not be needed.